towncrier.toml

```
@@ -4,6 +4,7 @@ filename = "CHANGELOG.md"
 start_string = "<!-- towncrier release notes start -->\n"
 underlines = ["", "", ""]
 title_format = "## [{version}](https://github.com/Substra/orchestrator/releases/tag/{version}) - {project_date}"
+issue_format = "[{issue}](https://github.com/Substra/orchestrator/pull/{issue})"
```

```suggestion
issue_format = "[#{issue}](https://github.com/Substra/orchestrator/pull/{issue})"
```
Missing a "#" to match perfectly ?